### PR TITLE
Fixed range() issue

### PR DESCRIPTION
**Pull request by mcculloh**
Range needs integer parameters
Simplified the color math
Pulled a set() call out of the hack loop

### CODE REVIEW

**Comment by mcculloh** [comment]
Hmm, looks like changes in the processing-video includes are not getting picked up correctly by ant in the build.

**Comment by mcculloh** [comment]
@jdf Did some code cleanup in an example

File: examples.py/Basics/Color/WaveGradient.py [hunk 1]
```diff
@@ -16,19 +16,20 @@ def setup():
...
+  j_del = 255.0 / (width + 50)
   for i in range(-75, height + 75):
     # Reset angle to 0, so waves stack properly
     angle = 0
     # Increasing frequency causes more gaps
     frequency += .006
     for j in range(width + 75):
-      py = i + sin(radians(angle)) * amplitude
+      sy = sin(radians(angle))
+      py = i+ int(sy*amplitude)
```
Comment (jdf) [inline examples.py/Basics/Color/WaveGradient.py]: Please auto-format.

Comment (mcculloh) [inline examples.py/Basics/Color/WaveGradient.py]: pep8 format?

**Comment by tildebyte** [comment]
I think he means

````py
py = i + int(sy * amplitude)
````

**Comment by mcculloh** [comment]
I discovered pep8 (pip install pep8) which is kinda neat. You run your program thru it and it gives you an update on formatting. 

For example, this file was using 2 space indents.

**Comment by jdf** [comment]
The Python mode editor has ctrl-T, which does pep8 formatting.

File: mode/examples/Basics/Color/WaveGradient/WaveGradient.pyde [hunk 2]
```diff
@@ -16,20 +16,21 @@ def setup():
...
+    j_del = 255.0 / (width + 50)
     for i in range(-75, height + 75):
         # Reset angle to 0, so waves stack properly
         angle = 0
         # Increasing frequency causes more gaps
         frequency += .002
         for j in range(0, width + 75):
-            py = i + sin(radians(angle)) * amplitude
+            sy = sin(radians(angle))
+            py = i + int( sy * amplitude )
```
Comment (jdf) [inline mode/examples/Basics/Color/WaveGradient/WaveGradient.pyde]: The spaces inside the parens don't look right to me.

**Comment by mcculloh** [comment]
Did more cleanup on the formatting. @jdf  @tildebyte anything else need review?

**Comment by n2v2rda2** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

n2v2rda2@naver.com 님께 보내신 메일 <Re: [processing.py] Fixed range() issue (#140)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

File: mode/examples/Basics/Color/WaveGradient/WaveGradient.pyde [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-Wave Gradient 
-by Ira Greenberg.    
+Wave Gradient
+by Ira Greenberg.
 
 Generate a gradient along a sin() wave.
 """
 amplitude = 30
-fillGap = 2.5
+fillGap = 2
```
Comment (jdf) [inline mode/examples/Basics/Color/WaveGradient/WaveGradient.pyde]: Why this change? Is the output identical?

Comment (mcculloh) [inline mode/examples/Basics/Color/WaveGradient/WaveGradient.pyde]: This was the original reason for the change. This is used later on in a call to range(). Python requires parameters to range() be integers. So the original would not even run for me.

Comment (mcculloh) [inline mode/examples/Basics/Color/WaveGradient/WaveGradient.pyde]: Probably should make it 

fillGap = 3

to have the range() loop below iterate over 1,2 instead of just 1. Visually the output is the same.

Comment (jdf) [inline mode/examples/Basics/Color/WaveGradient/WaveGradient.pyde]: Sounds good. As long as the output is the same, I'm happy.

**Comment by mcculloh** [comment]
@jdf --- uhh, did you want to pull this? I had completely forgotten about it...

**Comment by jdf** [comment]
Please confirm that the sketch is as formatted by ctrl-T in the Python Mode editor.

File: examples.py/Basics/Color/WaveGradient.py (hunk 1 above)
Comment (villares) [reply examples.py/Basics/Color/WaveGradient.py]: > pep8 format?

Cntrl/Cmmd + T is a Processing IDE "auto-pep8-formatter"
